ledger/src/integration-test/java/com/scalar/dl/ledger/service/LedgerServiceEndToEndTest.java

```diff
@@ -324,10 +327,14 @@ public static void setUpBeforeClass() throws Exception {
   }
 
   @AfterAll
-  public static void tearDownAfterClass() throws SchemaLoaderException {
+  public static void tearDownAfterClass() {
     storageAdmin.close();
-    SchemaLoader.unload(props, ledgerSchemaPath, true);
-    SchemaLoader.unload(props, databaseSchemaPath, true);
+    try {
```

Comment:
The tear-down sometimes failed due to the emulator issue, and it made the CI fail. However, the failure doesn't affect the next tests since the integration tests run locally in the CI, and this is the last. So, I changed it to just logging the warning.